rsts/deployment/security/security.rst

```diff
@@ -58,3 +58,56 @@ There are two options to fix this:
 
 #. Switch to a full external OAuth Provider (okta, GCP cloud identity, keycloak, Azure AD, etc.).
 #. Use a CA-certified SSL cert.
+
+********************************************************
+Running flyteadmin and flyteconsole on different domains
+********************************************************
+In some cases when flyteadmin and flyteconsole are running on different domains then you would need to allow the flyteadmin's domain to allow cross origin request from the flyteconsole's domain
+This can be done by changing the flyteadmin config in the following manner
+
+#. <flyte-admin-domain> is the domain which will be get request
+#. <flyte-console-domain> is the domain which will be sending the request as the originator
+#. <flyteconsole-ns> k8s namespace where your flyteconsole pod is running.
+#. <flyteadmin-ns> k8s namespace where your flyteadmin pod is running.
+#. Modify the flyteconsole deployment to use <flyte-admin-domain> as follows.
+
+   .. prompt:: bash
+
+      kubectl edit deployment flyteconsole -n <flyteconsole-ns>
+
+   .. code-block::
+
+      - env:
+        - name: ENABLE_GA
+          value: "true"
+        - name: GA_TRACKING_ID
+          value: G-0123456789
```

Comment:
These don't need to be here, right?

Reply:
I have put this in for the context so that the user understand where to put the new variable in. Let me know if there is some other way we can do this